lib/auth/auth.ts

```diff
@@ -205,7 +206,8 @@ function generateV4Headers(
     request.setHeader('host', request._headers.host);
     request.setHeader('x-amz-date', amzDate);
     request.setHeader('x-amz-content-sha256', payloadChecksum);
```

Comment:
Do we have the ticket related to the lack of payload SHA verification in CloudServer?

Reply:
I'll create one

Reply:
https://scality.atlassian.net/browse/S3C-8368